core/src/main/java/com/graphhopper/apache/commons/collections/IntFloatBinaryHeap.java

```diff
@@ -15,29 +15,26 @@
  */
 package com.graphhopper.apache.commons.collections;
 
-import com.graphhopper.coll.BinHeapWrapper;
-
 import java.util.Arrays;
 import java.util.NoSuchElementException;
 
 /**
  * This class is a partial copy of the class org.apache.commons.collections.BinaryHeap for
- * just the min heap and primitive, sorted double keys and associated int elements.
+ * just the min heap and primitive, sorted float keys and associated int elements.
  * <p>
  * The library can be found here: https://commons.apache.org/proper/commons-collections/
  */
-public class IntDoubleBinaryHeap implements BinHeapWrapper<Number, Integer> {
-
+public class IntFloatBinaryHeap {
```

Comment:
I renamed this from `IntDouble` to `IntFloat`, because this is what it actually uses. Shall we simply rename it to `(GH?)BinaryHeap`? Also I removed the `BinHeapWrapper` interface, because it looked like this was only needed for testing and I achieved the same with a testing interface.